### DISCUSSION TITLE
Git Storage - "error: unknown option 'signed=if-asked' usage: git push"

### DISCUSSION THREAD

**Discussion by GrayW** [requarks/wiki#3204] [Help / Questions] [answered]
Hello! 

I've been having a little trouble getting the Git Storage option to work as expected. I've resolved a few issues so far, but this one has me confused. I've tried searching in general and through existing issues and discussions, but I can't find anything that covers this issue.

Running 'git push' from the repo directory on the server manually pushes to GitHub perfectly fine, but a force sync or sync schedule results in this error: "error: unknown option 'signed=if-asked' usage: git push"

![wikijs-gitstorage-error](https://user-images.githubusercontent.com/6247450/110803097-c18e1c80-8276-11eb-917e-b57c6a505a03.png)

I'm sure there's a really simple global option I can set somewhere or something, but I just can't find anything. Any suggestions greatly received!

### Environment

**Install Type:** Linux [Using the docs](https://docs.requarks.io/install/linux)
**Wiki.js Version:** 2.5.170
**OS:** CentOS 7
**Git Version:** 1.8.3.1

**Comment by NGPixel** [comment, marked as answer]
Your version of git is too old and not supported. Read the [docs](https://docs.requarks.io/storage/git).

**Reply by GrayW** [reply]
Wow, apologies, I have no idea how I missed that. Thank you.
